src/lib/mapping.js

```diff
@@ -70,7 +70,8 @@ function mapTeamList (row) {
     description: row.description,
```

Comment:
That last sentence makes very much sense. I was going to wrap the data coming out of the ops call but we should really only care about the endpoints.